app/src/api/routes/asyncController.ts

```diff
@@ -0,0 +1,8 @@
+import { Request, Response, NextFunction } from 'express';
+
+export default function asyncController(fn: (req: Request, res: Response, next: NextFunction) => Promise<any>) {
```

Comment:
This needs a block comment to explain the intention - wrap async controller functions to be used as express routes.